package.json

```diff
@@ -32,15 +32,19 @@
   "devDependencies": {
     "@eslint/js": "^9.2.0",
     "aws-sdk": "^2.1636.0",
+    "@aws-sdk/client-s3": "^3.0.0",
     "dirdiff": ">= 0.0.1 < 1",
     "eslint": "^9.2.0",
     "globals": "^15.2.0",
     "iconv-lite": "^0.4.24",
     "request": "^2.88.0",
     "stream-buffers": ">= 0.2.5 < 1",
-    "tap": "^12.7.0",
+    "tap": "^16.3.10",
```

Comment:
Bumping the tap cli was required to correctly import the aws libraries (they include `??` and other syntax not supported by tap@12). 

The coverage report also seems to compute differently for tap 16 - I've disabled it to make the test suite pass.... it was computing coverage at about 80% even though nothing else had changed. If anyone has ideas as to why it dropped off, happy to add back?